index.html

```diff
@@ -270,87 +329,289 @@
+<dfn data-lt="FIPs">Fair Information Practices</dfn> ([=FIPs=]), a loose set of principles initially
+elaborated in the 1970s in support of individual [=autonomy=] in the face of growing concerns with databases. The
+[=FIPs=] generally assume that there is sufficiently little [=data processing=] taking place that any
+[=person=] will be able to carry out sufficient diligence to enable [=autonomy=] in their
+decision-making. Since they entirely offload the [=privacy labour=]
+to people and assume perfect, unlimited [=autonomy=], the [=FIPs=] do not forbid specific
+types of [=data processing=] but only place them under different procedural requirements.
+Such an approach is [=appropriate=] for [=parties=] that are processing data in the 1970s.
+
+One notable issue with procedural, self-governing approaches to privacy is that they tend to have the same
+requirements in situations where people find themselves in a significant asymmetry of
+power with a [=party=] — for instance a [=person=] using an essential service provided by a
+monopolistic platform — and those where people and [=parties=] are very much on equal
+footing, or even where the [=person=] may have greater power, as is the case with small
+businesses operating in a competitive environment. It further does not consider cases in
+which one [=party=] may coerce other [=parties=] into facilitating its [=inappropriate=]
+practices, as is often the case with dominant players in advertising or
+in content aggregation ([[?Consent-Lackeys]], [[?CAT]]).
+
+Reference to the [=FIPs=] survives to this day. They are often referenced as <em>transparency
+and choice</em>, which, in today's digital environment, is often a strong indication that
+[=inappropriate=] [=processing=] is being described.
 
-A <dfn data-lt="vulnerable">vulnerable person</dfn> is a [=person=] who may be unable to
-exercise sufficient self-determination in a [=context=]. Amongst other things, they should
-be treated with greater default privacy protections and may be considered unable to
-[=consent=] to various interactions with a system.
-People can be vulnerable for different reasons, for example because they are children,
-are employees with respect to their employers, are facing a steep asymmetry of power,
-are people in some situations of intellectual or psychological impairment, are
-refugees, etc.
 
-## Their Software Agents {#user-agents}
 
-The <dfn>user agent</dfn> acts as an intermediary between a [=person=] (its [=user=]) and the web. The
-[=user agent=] is <em>not</em> a [=context=] because it is expected to align fully with the
-[=person=] using it and operate exclusively in that [=person=]'s interest. It is <em>not</em> the
-[=first party=]. The [=user agent=] serves the [=person=] as a <dfn>trustworthy agent</dfn>:
+## Collective Governance {#collective}
+
+Privacy [=rules=] are socially negotiated and often contested ([[?Privacy-Contested]]). This makes
```

Comment:
I've made it more about the terms of the essentially contested paper. I think it relates to governance because it's really hard to govern if the participants don't agree on the rules (as we can see!). I also merged it with Viljoen's argument.